Formula/nmh.rb

```diff
@@ -0,0 +1,35 @@
+class Nmh < Formula
+  desc "Nmh, the new version of the MH mail handler"
+  homepage "http://www.nongnu.org/nmh/"
+  url "https://download.savannah.gnu.org/releases/nmh/nmh-1.6.tar.gz"
+  sha256 "29338ae2bc8722fe8a5904b7b601a63943b72b07b6fcda53f3a354edb6a64bc3"
+
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}", "--libdir=#{libexec}",
+                          "--with-cyrus-sasl",
+                          "--with-tls"
+    system "make", "install"
+  end
+
+  test do
+    Dir.mkdir(testpath/"Mail")
+    Dir.mkdir(testpath/"Mail/inbox")
+    (testpath/".mh_profile").write "Path: Mail"
+    (testpath/"Mail/inbox/1").write <<-EOM.undent
+      From: Mister Test <test@example.com>
+      To: Mister Nobody <nobody@example.com>
+      Date: Tue, 5 May 2015 12:00:00 -0000
+      Subject: Hello!
+
+      How are you?
+    EOM
+    ENV["TZ"] = "GMT"
+    output = `#{bin}/scan -width 80`
```

Comment:
Unfortunately I don’t think we documented this somewhere. It’s used in a lot of formulae (see e.g. [`aardvark_shell_utils.rb`](https://github.com/Homebrew/homebrew-core/blob/cb886e6d5b8255ad8139fce2a25b1afb9ea15589/Formula/aardvark_shell_utils.rb#L27-L29), [`cgrep.rb`](https://github.com/Homebrew/homebrew-core/blob/1668cbc72bf4f5d38e7316774251f266bfc2f8f7/Formula/cgrep.rb#L34-L39), [`ppss.rb`](https://github.com/Homebrew/homebrew-core/blob/2c5572c61df26d7915af8ee796bad3dfea66ed90/Formula/ppss.rb#L14)). This is a function [defined in Homebrew](https://github.com/Homebrew/brew/blob/028d155e97014076adf4e4dbc7c888fd871de321/Library/Homebrew/formula_assertions.rb#L27) that’s an equivalent of `` `…` `` but asserts that the command ran successfully.

(note to myself or anyone reading this: document `shell_output` somewhere)